nyxtools/nyx_raster_flyer.py

```diff
@@ -0,0 +1,167 @@
+import getpass
+import grp
+import logging
+import os
+import time as ttime
+
+from ophyd.sim import NullStatus
+from ophyd.status import SubscriptionStatus
+
+from .flyer_eiger2 import NYXEiger2Flyer
+
+logger = logging.getLogger(__name__)
+
+INTERNAL_SERIES = 0
+INTERNAL_ENABLE = 1
+EXTERNAL_SERIES = 2
+EXTERNAL_ENABLE = 3
+
+
+class NYXRasterFlyer(NYXEiger2Flyer):
+    def __init__(self, vector, zebra, detector=None) -> None:
+        super().__init__(vector, zebra, detector)
+        self.name = "NYXRasterFlyer"
+
+    def kickoff(self):
+        print("kickoff")
+        # still need to stage the detector in lsdc
+
+        def armed_callback(value, old_value, **kwargs):
+            if old_value == 0 and value == 1:
+                return True
+            return False
+
+        zebra_status = SubscriptionStatus(self.zebra.pc.arm.output, armed_callback, run=False)
+        self.zebra.pc.arm_signal.put(1)
+        zebra_status.wait()
```

Comment:
`kickoff` should return as quickly as possible, not block until it is done.  You should be returning `zebra_status` not waiting on it.

Reply:
I was told that the zebra arming is a negligible amount of time, I am waiting here to ensure that there can not be an issue. Is the kickoff not supposed to be kicking off the vector movement? Would you rather I returned the status and then waited and began the movement? That would be a significant deviation from what the mxtools kickoff was doing, but it shouldn't be a problem if you want me to add it.

Reply:
I was told that the zebra arming is a negligible amount of time, I am waiting here to ensure that there can not be an issue. Is the kickoff not supposed to be kicking off the vector movement? Would you rather I returned the status and then waited and began the movement? That would be a significant deviation from what the mxtools kickoff was doing, but it shouldn't be a problem if you want me to add it.